eth_utils/logging.py

```diff
@@ -60,10 +63,7 @@ def _use_logger_class(logger_class: Type[logging.Logger]) -> Iterator:
         logging.setLoggerClass(original_logger_class)
 
 
-TLogger = TypeVar("TLogger", bound=logging.Logger)
-
-
-def get_logger(name: str, logger_class: Type[TLogger] = None) -> TLogger:
+def get_logger(name: str, logger_class: Type[logging.Logger] = None) -> logging.Logger:
```

Comment:
Can you update the typing for this to use:

```python
TLogger = TypeVar("TLogger", bound=logging.Logger)
def get_logger(name: str, logger_class: Type[TLogger]) -> TLogger;
    ...
```

I think it is more correct and will remove the need to use a `cast` in `get_extended_debug_logger`

Reply:
Yeah, this is what I've tried originally, but this solution requires that I cast the returned loggers on line 68, 84 to `TLogger` - otherwise mypy complains with 

> eth_utils/logging.py:68: error: Incompatible return value type (got "Logger", expected "TLogger")
> eth_utils/logging.py:84: error: Incompatible return value type (got "Logger", expected "TLogger")

 But maybe that's preferable to performing the cast in `get_extended_debug_logger`?

Reply:
I think that is indeed preferrable (to cast things to `TLogger`)